src/test/shell/bazel/check_external_files_test.sh

```diff
@@ -81,8 +78,8 @@ setup_remote() {
 
   mkdir main
   cd main
-  cat >> "WORKSPACE" <<EOF
-load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+  cat >> "MODULE.bazel" <<EOF
```

Comment:
You need setup_module_dot_bazel since it's create a new workspace directory ("main).

Reply:
done, thanks!